modules/iam-identity/src/main/java/com/ibm/cloud/platform_services/iam_identity/v1/IamIdentity.java

```diff
@@ -91,7 +96,7 @@ public class IamIdentity extends BaseService {
   /**
    * Default service endpoint URL.
    */
-  public static final String DEFAULT_SERVICE_URL = "https://iam.cloud.ibm.com";
+  public static final String DEFAULT_SERVICE_URL = "https://iam.test.cloud.ibm.com";
```

Comment:
This change should be reverted.  We should always use the production endpoint URL in the generated code.  My guess is that you used the test branch version of the API definition when generating the code, and you should be using the production version.

Reply:
this is fixed.